napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -876,8 +1006,15 @@ def setup_ui(self):
         self.packages_filter.setPlaceholderText(trans._("filter..."))
         self.packages_filter.setMaximumWidth(350)
         self.packages_filter.setClearButtonEnabled(True)
+        self.refresh_button = QPushButton(trans._('Refresh'), self)
```

Comment:
Can we add a small tooltip here detailing what things this does refresh? E.g. does this re-query npe2api?

Reply:
Will add the tooltip, thanks!

Reply:
Added!